CPAC/utils/configuration/yaml_template.py

```diff
@@ -119,8 +120,16 @@ def dump(self, new_dict, parents=None):
             # Insert automatically-changed value in original dict
             if freesurfer_extraction:
                 new_dict = set_nested_value(
-                    new_dict, ['surface_analysis', 'freesurfer', 
+                    new_dict, ['surface_analysis', 'freesurfer',
                     'run_reconall'], False)
+            try:
+                self.get_nested(new_dict,
+                                ['surface_analysis', 'freesurfer',
+                                 'ingress_reconall'])
+            except KeyError:
+                new_dict = set_nested_value(new_dict,
+                                            ['surface_analysis', 'freesurfer',
+                                             'ingress_reconall'], False)
```

Comment:
I know what's here was my idea, but when responding [here](https://github.com/FCP-INDI/C-PAC/pull/1890#issuecomment-1448753216), I realized we should actually be pulling in the imported value instead of always setting an imported value to `False`

```suggestion
            ingress_keys = ['surface_analysis', 'freesurfer', 'ingress_reconall']
            try:
                self.get_nested(new_dict, ingress_keys)
            except KeyError:
                new_dict = set_nested_value(new_dict, ingress_keys,
                                            self.get_nested(self._dict, ingress_keys))
```